code/headers/hardware_usart.hpp

```diff
@@ -41,7 +45,7 @@ namespace r2d2 {
         /// used for char arrays to quickly send more than one byte
         /// In practice useing this fuction is more stable than repeated use of
         /// send_byte() Especially when the values are repeated.
-        hardware_usart_c &operator<<(const char *c) ;
+        hardware_usart_c &operator<<(const char *c);
```

Comment:
Could a overload for ostream be used here? That way, the conversion operations it provides would be available